test/Unit/App/Attribute/MethodDeprecationTest.php

```diff
@@ -55,6 +56,46 @@ public function test(): void
         $this->assertSame('test-deprecation-reason', $attribute->deprecationReason);
     }
 
+    public function testToArray(): void
+    {
+        $class = new class {
+            #[MethodDeprecation(
+                sunset: '2038-01-01',
+                link: 'test-link',
+                deprecationReason: 'test-deprecation-reason',
+                rel: 'test-rel',
+                type: 'test-type',
+            )]
+            public function test(): void
+            {
+            }
+        };
+
+        $reflectionClass = new ReflectionClass($class);
+        $attributes      = $this->getAttributes($reflectionClass);
+
+        $this->assertIsArray($attributes);
```

Comment:
```suggestion
        $this->assertNotEmpty($attributes);
```
psalm: RedundantCondition: Type array<array-key, mixed> for $attributes is always array<array-key, mixed>